packages/unimodules-test-core/android/src/main/java/expo/modules/test/core/ModuleMockHolder.kt

```diff
@@ -0,0 +1,11 @@
+package expo.modules.test.core
+
+import expo.modules.core.interfaces.services.EventEmitter
+import expo.modules.kotlin.AppContext
+
+data class ModuleMockHolder<T>(
```

Comment:
Once again I'm not sure about the name suggestion, but is it possible to limit generic to the actual `MockedModule` implementation? 🤔 
```suggestion
data class MockedModuleHolder<T extends something?>(
```

Reply:
> is it possible to limit generic to the actual MockedModule implementation? 🤔

Unfortunately not.